spec/index.md

```diff
@@ -598,8 +599,102 @@ MISSING
 
 ### 5.1.1.13. Event types
 
-An *event type* is the type denotated by the name of an [`event` item].
+An *event type* is the type denoted by the name of an [`event` item].
 
+## 6. Data Layout
+
+There are three places where data can be stored on the EVM:
+
+- stack
+- storage
+- memory
```

Comment:
Should we count `calldata` as well?

Reply:
I initially had it in there, but took it out since there's not a whole lot to say about it. The data will always be laid out according to the contract ABI spec.